lib/pbench/test/functional/server/test_datasets.py

```diff
@@ -462,16 +462,23 @@ class TestIndexing:
     def test_details(self, server_client: PbenchServerClient, login_user):
         """Check access to indexed data
 
-        Perform a GET /datasets/details/{id} to be sure that basic run data
-        has been indexed and is available.
+        Perform a GET /datasets/details/{id} to be confirm whether indexed run
+        data is available.
         """
         print(" ... checking dataset RUN index ...")
         datasets = server_client.get_list(
-            metadata=["dataset.metalog.pbench,server.archiveonly"], owner="tester"
+            metadata=["dataset.metalog.pbench"], owner="tester"
         )
         for d in datasets:
-            print(f"\t... checking run index for {d.name}")
-            indexed = not d.metadata["server.archiveonly"]
+
+            # Query the dataset's indices: if the returned object is empty,
+            # there are none.
+            indices = server_client.get(API.DATASETS, {"dataset": d.resource_id})
+            indexed = bool(indices.json())
+            print(
+                f"\t... checking run details for {d.name} "
+                f"({'' if indexed else 'not '} indexed)"
```

Comment:
I think you've got an extra space between the closing brace and `indexed` in line 480.

Reply:
There's a space at the end of the string containing `not`, and there's a space in the outer string in front of `indexed`, so that's going to be two spaces in a row in that case; but, more obviously, in the other case, without the `not`, there's going to be a space between the opening paren and `indexed`.

Reply:
These are not the spaces you're looking for. Go about your business.

Reply:
Oops.